packages/kit/test/apps/basics/src/routes/selection/_tests.js

```diff
@@ -0,0 +1,32 @@
+import * as assert from 'uvu/assert';
+
+/** @type {import('test').TestMaker} */
+export default function (test) {
+	test('reset selection', '/selection/a', async ({ page, clicknav }) => {
+		assert.equal(
+			await page.evaluate(() => {
+				const range = document.createRange();
+				range.selectNodeContents(document.body);
+				const selection = getSelection();
+				if (selection) {
+					selection.removeAllRanges();
+					selection.addRange(range);
+					return selection.rangeCount;
+				}
+				return 0;
+			}),
+			1
+		);
+		await clicknav('[href="/selection/b"]');
+		assert.equal(
```

Comment:
i'm worried this might introduce flakiness, which is why you sometimes needed the timeout. I wonder if you need to wait until the selection is present instead of simply checking that it is